.github/workflows/update_submodule.yml

```
@@ -44,7 +44,7 @@ jobs:
 
       - name: Update submodule
         run: |
-          git -C "ext/${REPO_NAME}" fetch origin "${TAG_NAME}"
+          git submodule update --remote "ext/${REPO_NAME}"
```

Same change. Same objection. Why is this different than a fetch?

---

The difference between the two commands is not entirely clear. However, it was determined from local testing...by setting up the repository and checkout with the same set of commands used by GitHub Actions...that the `fetch` command does not work while the `submodule update --remote` command does:

```
❯ git -C ext/cli fetch origin v6.1.0
remote: Enumerating objects: 191, done.
remote: Counting objects: 100% (191/191), done.
remote: Compressing objects: 100% (92/92), done.
remote: Total 140 (delta 108), reused 74 (delta 46), pack-reused 0
Receiving objects: 100% (140/140), 46.81 KiB | 4.25 MiB/s, done.
Resolving deltas: 100% (108/108), completed with 41 local objects.
From https://github.com/phylum-dev/cli
 * tag               v6.1.0     -> FETCH_HEAD

❯ git -C ext/cli tag

❯ git -C ext/cli checkout v6.1.0
error: pathspec 'v6.1.0' did not match any file(s) known to git

❯ git submodule update --remote ext/cli
From https://github.com/phylum-dev/cli
 * [new tag]         v6.1.0     -> v6.1.0
Submodule path 'ext/cli': checked out '60b3338579e2487a6d3f4be3fcf8287b028e962a'

❯ git -C ext/cli tag
v6.1.0

❯ git -C ext/cli checkout v6.1.0
Previous HEAD position was 60b3338 Bump dependencies (#1353)
HEAD is now at 67dbfd7 Bump to v6.1.0 (#1348)
```

---

The difference is made clear in your output. The `git fetch` command as written stores the commit in `FETCH_HEAD`

```
From https://github.com/phylum-dev/cli
 * tag               v6.1.0     -> FETCH_HEAD
```

While the update stores the commit in a new tag.

```
From https://github.com/phylum-dev/cli
 * [new tag]         v6.1.0     -> v6.1.0
```

This was my mistake thinking that we could use the shorter command. The corrected fetch command is this one:

```suggestion
          git -C "ext/${REPO_NAME}" fetch origin "+refs/tags/${TAG_NAME}:refs/tags/${TAG_NAME}"
```

Or something more generic could be used

```suggestion
          git -C "ext/${REPO_NAME}" fetch --all --tags
```